tests/test_gdal_subsetter.py

```diff
@@ -77,18 +77,26 @@ def test_unknown_file_format(self, mock_download):
         stac_item.add_asset('data', Asset('url.com/file.xyz', roles=['data']))
         stac_catalog.add_item(stac_item)
 
-        harmony_message = Message({'accessToken': 'fake-token',
-                                   'callback': 'https://example.com',
-                                   'format': {'mime': 'image/png'},
-                                   'sources': [{'collection': 'C1234-XYZ'}],
-                                   'stagingLocation': 's3://example-bucket',
-                                   'user': 'ascientist'})
-
-        adapter = HarmonyAdapter(harmony_message, catalog=stac_catalog,
-                                 config=self.config)
-
-        with self.assertRaises(UnknownFileFormatError) as context_manager:
+        harmony_message = Message(
+            {
+                'accessToken': 'fake-token',
+                'callback': 'https://example.com',
+                'format': {'mime': 'image/png'},
+                'sources': [{'collection': 'C1234-XYZ'}],
+                'stagingLocation': 's3://example-bucket',
+                'user': 'ascientist'
+            }
+        )
+
+        adapter = HarmonyAdapter(
+            harmony_message, catalog=stac_catalog, config=self.config
+        )
+
+        with self.assertRaises(UnsupportedFileFormatError) as context_manager:
```

Comment:
Swapping out the exception type (and moving the assertions about the messages to the right place) is the only substantive change to these tests. The rest is formatting stuff.